Sources/MockableMacro/Extensions/DeclModifierListSyntax+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//
+//
+//  Created by Nayanda Haberty on 29/5/24.
+//
+import SwiftSyntax
+
+extension DeclModifierListSyntax {
```

Comment:
I would not create an extension for this, I don't see this being reused elsewhere in the macro implementation. Lets move this to the `Requirements` `init(_:)` for now and we can create an extension if we need similar logic later.

Reply:
That makes sense. Will refactor